rerun_py/rerun_sdk/rerun/log_deprecated/rects.py

```diff
@@ -138,7 +138,7 @@ def log_rects(
     entity_path:
         Path to the rectangles in the space hierarchy.
     rects:
-        Nx4 numpy array, where each row is [x, y, w, h], or some format you pick with the `rect_format` argument.
+        Nx4 numpy rects, where each row is [x, y, w, h], or some format you pick with the `rect_format` argument.
```

Comment:
similar, it's a single Nx4 numpy _array_. It's just that `N` is the number of rects. So maybe
```suggestion
        Nx4 numpy array for N rects, where each row is [x, y, w, h], or some format you pick with the `rect_format` argument.
```